action.yml

```diff
@@ -0,0 +1,28 @@
+name: 'File Existence and Reader'
```

Comment:
Let's use some action name such as "Action validation checks"
Final name we can take Pulkit's inputs later on